backup-recovery.adoc

```diff
@@ -1,26 +1,101 @@
+
+**Account aging** establishes the age of your payment accounts. To reduce fraud, newer payment accounts have lower trading limits.
+
+To include account aging with your account details import, replace the `btc_mainnet/keys/sig.key` file with the `sig.key` file from your backup. Be sure you don't have any open offers, trades, or disputes before changing `sig.key`, otherwise you won't be able to respond to those activities.
+
+If you don't have an accounts export file, but you have a full backup of the data directory, you can still fully restore your accounts details and accounts aging by manually copying these files to your data directory from a backup:
+
+* `btc_mainnet/db/AccountAgeWitnessStore` (salt)
+* `btc_mainnet/keys/sig.key` (key used to sign p2p messages)
+* `btc_mainnet/keys/UserPayload` (account details)
+
+**Again, please make sure you don't have any open offers, trades, or disputes before changing `sig.key`.**
+
+IMPORTANT: Before manually making any changes to Bisq data files, please remember to back up the current state of your Bisq data directory and make sure the Bisq program is closed.
+
+== Tor State
+
+Bisq runs as a Tor hidden service to protect your privacy.
+
+As you do trades, you'll build _local reputation_ whereby Bisq will show you how many trades you've completed with other peers. The idea is that if one trade with someone went well, it's reasonable to expect your next trade with that person will go well too, creating a rudimentary sort of reputation while maintaining anonymity. This mechanism is tied to your onion address.
+
+If you want to migrate a particular onion address, you can replace the `btc_mainnet/tor/hiddenservice` folder in your data directory with the one from your backup.
+
+If you want to change your onion address, just delete the `btc_mainnet/tor/hiddenservice`. The next time you start Bisq, you'll have a different onion address.
+
+**Make sure you don't have any open offers, trades, or disputes before changing your onion address, as doing so will make you unreachable.**
+
+IMPORTANT: Before manually making any changes to Bisq data files, please remember to back up the current state of your Bisq data directory and make sure the Bisq program is closed.
+
+== Wallet Data
+
+Your wallet data is stored in `btc_mainnet/wallet`.
+
+Wallet issues (e.g., incorrect balance showing, corrupted/out-of-sync SPV file) can often be solved by resyncing the SPV chain and restarting the Bisq program. You can do this by clicking the `Delete SPV file and resync` button on the `Network info` tab of the `Settings` screen. If you can't access this screen, you can just delete the `bisq.spvchain` file in `btcmainnet/wallet` (just be advised that you will need to manually restart Bisq twice to complete the resync).
```

Comment:
Re-sync can take a while and CPU will be at 100% for that time. The more transactions you had and the older the wallet is the longer it takes. Bisq UI is not responsive at that time as the wallet resync takes up all CPU resources.